app/components/cases/parliament/newest-documents-panel.js

```diff
@@ -0,0 +1,58 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+export default class CasesParliamentNewestDocumentsPanelComponent extends Component {
+  @service store;
+
+  @tracked piecesBySubcase = {};
+
+  constructor() {
+    super(...arguments);
+    this.loadData.perform();
+  }
+
+  loadData = task(async () => {
+    const subcase = await this.args.parliamentFlow.parliamentSubcase;
+    const submissionActivities = await this.store.queryAll(
+      'parliament-submission-activity',
+      {
+        'filter[parliament-subcase][:id:]': subcase.id,
+      }
+    );
+
+    const submittedPieces = await this.store.queryAll('submitted-piece', {
+      'filter[parliament-submission-activity][:id:]': submissionActivities
+        .map((x) => x.id)
+        .join(','),
+      sort: '-parliament-submission-activity.start-date',
+      include: 'piece',
+    });
+
+    const uniquePieces = removeDuplicatePiece(submittedPieces.slice());
```

Comment:
I'm wondering what the removeDuplicatePieces will do in case a `file` was changed and re-submitted. 
It's probably fine since the `piece` itself is the same, and the `file` isn't loaded until someone actually clicks on the `piece` to go to the document-viewer, but we should definitely verify this during testing